modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/portlet/action/EditAccountUserMVCActionCommand.java

```diff
@@ -84,31 +85,55 @@ private void _editAccountUser(User accountUser, ActionRequest actionRequest)
 
 		accountUserContact.setPrefixListTypeId(
 			_getListTypeId(
-				accountUser.getCompanyId(), actionRequest,
-				"prefixListTypeValue", ListTypeConstants.CONTACT_PREFIX));
+				accountUser, "prefixListTypeValue", actionRequest,
+				ListTypeConstants.CONTACT_PREFIX));
 		accountUserContact.setSuffixListTypeId(
 			_getListTypeId(
-				accountUser.getCompanyId(), actionRequest,
-				"suffixListTypeValue", ListTypeConstants.CONTACT_SUFFIX));
+				accountUser, "suffixListTypeValue", actionRequest,
+				ListTypeConstants.CONTACT_SUFFIX));
 
 		_contactLocalService.updateContact(accountUserContact);
 	}
 
 	private long _getListTypeId(
-		long companyId, PortletRequest portletRequest, String parameterName,
-		String type) {
+			User accountUser, String parameterName,
```

Comment:
as we have to touch the PR (other comment) can we rename `accountUser` to just `user`? In case please remember also to sort again the params to have it as last